### DISCUSSION TITLE
registering more commands

### DISCUSSION THREAD

**Discussion by michael-dm** [edbrdi/n8n-nodes-discord#16] [Ideas]
Hello and great work 💪

Do you think you could provide a way to register our own commands along with de defaults '/logs, /test, clear' ?

**Comment by edbrdi** [comment]
Hello, thank you, it could be a nice feature. Do you have any use case examples? How would you see the implementation?

**Comment by michael-dm** [comment]
Well I suppose most of us will use this node to register commands in a Discord server. Right now users only have auto-completion for the 3 default commands. So use case is "having auto-completion for our custom commands too".

For implementation, maybe we could tick a checkbox in trigger node, with field to write command description etc.
Then the module could push commands info to Discord on workflow save for instance.

What do you think ?

**Comment by edbrdi** [comment]
I need to do some testing to see how I can manage new commands registration with the old ones. You're not the first one to ask for such feature so I will definitely dig into it.

**Comment by edbrdi** [comment]
"command trigger" is now available through the release 0.5.0
